packages/web-components/src/components/va-alert/va-alert.tsx

```diff
@@ -28,6 +28,12 @@ export class VaAlert {
    */
   @Prop() backgroundOnly: boolean = false;
 
+  /**
+   * This only takes effect when background-only is true. If true, the background-only alert will
+   * include an icon.
+   */
+  @Prop() showIconInBackgroundOnly: boolean = false; 
```

Comment:
I would like for this prop to be shorter if possible, but I'm not sure about what would be better. It might help if we had explicit answers to this question:

Do we want `backgroundOnly` alerts to have icons by default?
  - This changes the default behavior, and 10 alerts would be impacted in `vets-website` (none in `content-build`
    ![image](https://user-images.githubusercontent.com/2008881/144111811-90527782-9de6-4d36-9283-ffe8c1d89f1e.png)

Based on the change it seems like the design allows for a `backgroundOnly` alert to have an icon or not. Liz's comments on the original issue suggest that the preference among most teams _might_ be for the icon to already be there. If this is the case and we want to encourage that, it may make sense to switch the default behavior to include an icon, and give teams a way to opt out with a prop like `text-only` or something.

Reply:
If teams have a preference for using an icon in background-only alerts, then I agree with you that we should change the default behavior to display the icon. Should I go ahead and make these changes or should this question be directed to someone else?

Reply:
I actually think it would be good not to include the icon by default, as most of the time this alert is used without an icon. That could change in time, though.

Reply:
I reached out to Kevin, and he said the default behavior should be to not display the icon unless a developer decides otherwise. I've updated the prop name to `showIcon`.